Makefile

```
@@ -60,7 +60,7 @@ testdata:
 locales: 
 	msgfmt -o locale/ja/LC_MESSAGES/loris.mo locale/ja/LC_MESSAGES/loris.po
 	npx i18next-conv -l ja -s locale/ja/LC_MESSAGES/loris.po -t locale/ja/LC_MESSAGES/loris.json
-	msgfmt -o locale/hi/LC_MESSAGES/loris.mo locale/hi/LC_MESSAGES/loris.po
+	msgfmt -o locale/hi/LC_MESSAGES/loris.mo locale/ja/LC_MESSAGES/loris.po
```

Was this intentionally changed or it's a typo ? From previous `hi` to `ja`
```suggestion
	msgfmt -o locale/hi/LC_MESSAGES/loris.mo locale/hi/LC_MESSAGES/loris.po
```